src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using SFA.DAS.ProviderApprenticeshipsService.Infrastructure.Configuration;
+using FluentAssertions;
+using SFA.DAS.ProviderApprenticeshipsService.Web.Models;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class HomeControllerTest
+    {
+        private Web.Controllers.HomeController _sut;
+
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+
+        public void ShouldReturnValidViewModel(bool useDfESignIn)
+        {
+            // arrange
+            _sut = new Web.Controllers.HomeController(new ProviderApprenticeshipsServiceConfiguration
+            {
+                UseDfESignIn = useDfESignIn
+            });
+
+            // sut
+            var result = _sut.Index();
+
+            var vr = result as ViewResult;
+
+            // assert
+            vr.Should().NotBeNull();
+
+            var vm = vr.Model as HomeViewModel;
+            vm.Should().NotBeNull();
```

Comment:
Should we also assert that the flag is set correctly on the view model?

Reply:
Also, added Assertion to check the UseDfESignIn flag value in the view model.